src/pkg/build/builder_remote.go

```diff
@@ -155,9 +168,17 @@ func (rb *RemoteBuilder) streamOutput(ctx context.Context, url string) (err erro
 }
 
 // doBuildRequest creates a new build on a Remote Build Service
-func (rb *RemoteBuilder) doBuildRequest(ctx context.Context, d Definition) (rd ResponseData, err error) {
+func (rb *RemoteBuilder) doBuildRequest(ctx context.Context, d Definition, libraryRef string) (rd ResponseData, err error) {
```

Comment:
the `rb` var with method `doBuildRequest` has definition and library ref stored, we could just do `rb.definition` instead of passing then as an argument to the func

Reply:
They are different things. `rb.ImagePath` can refer to a local image path or a Library reference. The `libraryRef` passed to `doBuildRequest()` is always a Library reference (or `""`, in which case RB will auto-generate a Library reference.